src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -3662,9 +3677,17 @@ private static bool CanReuseBracketedParameterList(CSharp.Syntax.BracketedParame
                             {
                                 break;
                             }
-                            else if (this.CurrentToken.Kind == SyntaxKind.CommaToken || this.IsPossibleParameter())
+                            else if (this.CurrentToken.Kind == SyntaxKind.CommaToken ||
+                                     this.IsPossibleParameter())
```

Comment:
This is code style change. Wouldn't it be more appropriate to move that to separate commit?